src/main/java/uk/gov/hmcts/reform/sscs/validators/SscsCaseValidator.java

```diff
@@ -247,6 +251,13 @@ private Boolean doesAddressLine1Exist(Address address) {
         return false;
     }
 
+    private Boolean doesAddressLine2Exist(Address address) {
+        if (address != null) {
+            return address.getLine1() != null;
```

Comment:
And this line is looking at address line 1 but the method states address line 2

Reply:
copy paste error